PhysicsTools/PatAlgos/python/recoLayer0/bTagging_cff.py

```diff
@@ -12,12 +12,20 @@
   , 'inclusiveSecondaryVertexFinderFilteredNegativeTagInfos'
     # new candidate-based framework (supported with RECO/AOD/MiniAOD)
   , 'pfImpactParameterTagInfos'
+  , 'pfImpactParameterTagInfosAK8'
+  , 'pfImpactParameterTagInfosCA15'
   , 'pfSecondaryVertexTagInfos'
   , 'pfSecondaryVertexNegativeTagInfos'
   , 'pfInclusiveSecondaryVertexFinderTagInfos'
+  , 'pfInclusiveSecondaryVertexFinderTagInfosAK8'
```

Comment:
I do not get it, why do we need these? Why named TagInfos with the jetalgo/cone in it? PAT is supposed to take care and adjust the sequence... what is special about it? the enlarged deltaR? why do you want to enlarge deltaR?

@ferencek I do not like this even for BTV... what's the point of naming it AK or CA ? if it is just a matter of size, it should not have the AK/CA suffix.